docs/README.md

```diff
@@ -25,6 +23,20 @@
     <img src="https://img.shields.io/badge/twitter-%40preCICE__org-1da1f2.svg" alt="Twitter">
 </a>
 
+**Build Status**  
+<a style="text-decoration: none" href="https://travis-ci.org/precice/precice" target="_blank">
+    <img src="https://travis-ci.org/precice/precice.svg?branch=develop" alt="Build status">
+</a>
+<a style="text-decoration: none" href="https://travis-ci.org/precice/systemtests" target="_blank">
+    <img src="https://img.shields.io/travis/precice/systemtests.svg?label=system%20tests&style=flat" alt="Build status">
+</a>
+
+**Code Quality**  
+[![CodeFactor](https://www.codefactor.io/repository/github/precice/precice/badge)](https://www.codefactor.io/repository/github/precice/precice)
```

Comment:
All of them are "code quality", so (if possible), set the name of the tool as badge name.